nav2_bringup/launch/navigation_launch.py

```diff
@@ -42,7 +42,7 @@ def generate_launch_description():
                        'smoother_server',
                        'planner_server',
                        'behavior_server',
-                       'bt_navigator',
+                       'navigator_server',
```

Comment:
Revert. Please do not make unnecessary changes

Reply:
You made a bunch of changes related to this; all of which must be reversed before I can review again